packages/core/src/factories.ts

```diff
@@ -0,0 +1,9 @@
+import { UserSession } from "@esri/arcgis-rest-auth";
+import { HubSession } from "./HubSession";
+
+export async function buildHubSession(
```

Comment:
Can drop `async` since no `await`.

Do we want to add comments/JSDoc here and all method class members for consistency?